app/models/physical_server.rb

```diff
@@ -6,6 +6,7 @@ class PhysicalServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::PhysicalInfraManager"
 
   default_value_for :enabled, true
+  has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```

Comment:
Julian, is it ok that this is a repeat of commits in 14026?  When this is applied these commits will be skipped correct?  Or should we regenerate the PR when 14026 is approved?